packages/maps/src/components/result/GoogleMapMarker.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { MarkerWithLabel } from 'react-google-maps/lib/components/addons/MarkerWithLabel';
+import { InfoWindow, Marker } from 'react-google-maps';
+
+import types from '@appbaseio/reactivecore/lib/utils/types';
+import { setMarkerOnTop, setOpenMarkers } from '@appbaseio/reactivecore/lib/actions';
+import { connect } from '@appbaseio/reactivesearch/lib/utils';
+
+import { MapPin, MapPinArrow, mapPinWrapper } from './addons/styles/MapPin';
+
+class GoogleMapMarker extends React.Component {
+	shouldComponentUpdate(nextProps) {
```

Comment:
We're using `shouldComponentUpdate` but it may have UI issues when someone uses the `state` in the `renderData`.
Can we add one more prop by which we can bypass this behavior?
We're using `strict` prop in `StateProvider`.

Reply:
@bietkul what could be the issues, when someone uses `state` in `renderData`?